### PR TITLE
Add more documentation about worksheets.

### LINKED ISSUE #1935: Add better documentation for worksheets
We should create a separate section that explains better what worksheets are and explain all the possibilities like for example adding imports.

### PR DESCRIPTION
**Pull request by ckipp01**
Closes #1935

### CODE REVIEW

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+             |""".stripMargin
+      case _ =>
+        "as a comment as the end of the line." -> "hover on the comment to expand."
+    }
+
+    s"""|## Worksheets
+        |
+        |Worksheets are a great way to explore an api, try out an idea, or code
+        |up an example and quickly see the evaluated expression or result. Behind
+        |the schens worksheets are powered by the great work done in
```
Comment (tgodzik) [inline metals-docs/src/main/scala/docs/WorksheetModifier.scala:42]:
```suggestion
        |the scenes worksheets are powered by the great work done in
```

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        |[mdoc](https://scalameta.org/mdoc/).
+        |
+        |### Getting started with Worksheets
+        |
+        |To get started with a worksheet you can either use the `metals.new-scala-file`
+        |command and select *Worksheet* or create a file called `*.worksheet.sc`.
+        |This format is important as it is what tells Metals that it's meant to be
```
Comment (tgodzik) [inline metals-docs/src/main/scala/docs/WorksheetModifier.scala:49]:
```suggestion
        |This format is important as this is what tells Metals that it's meant to be
```
?

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        |treated as a worksheet and not just a Scala script. Where you create the
```
Comment (tgodzik) [inline metals-docs/src/main/scala/docs/WorksheetModifier.scala:50]: `It also matters where you create the worksheet` ?

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        |script also matters. If you'd like to use classes and values from your
+        |project, you need to make sure the worksheet is created inside of your `src`
+        |directory. You can still create a worksheet in other places, but you will
+        |only have access to the standard library.
```
Comment (tgodzik) [inline metals-docs/src/main/scala/docs/WorksheetModifier.scala:54]: `or the local library dependencies` ?

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala (hunk 1 above)
Comment (ckipp01) [reply metals-docs/src/main/scala/docs/WorksheetModifier.scala:42]: Fixed 👍

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala (hunk 2 above)
Comment (ckipp01) [reply metals-docs/src/main/scala/docs/WorksheetModifier.scala:49]: Changed.

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala (hunk 3 above)
Comment (ckipp01) [reply metals-docs/src/main/scala/docs/WorksheetModifier.scala:50]: I think both are actually correct.

File: metals-docs/src/main/scala/docs/WorksheetModifier.scala (hunk 4 above)
Comment (ckipp01) [reply metals-docs/src/main/scala/docs/WorksheetModifier.scala:54]: A good catch. I'll add this in.

**Pull request merged**